app-gateway.tf

```diff
@@ -24,7 +24,7 @@ locals {
 }
 
 module "appGw" {
-  source            = "git@github.com:hmcts/cnp-module-waf?ref=master"
+  source            = "git@github.com:hmcts/cnp-module-waf?ref=ccd/CHG0033576"
```

Comment:
Point WAF to detection mode brance due to false positives

Reply:
This change has to be approve by security

Reply:
It's already like this in production @qzhou-hmcts as per DevOps instructions - this is to make sure it's not reverted back
If we don't have this and our master branch is triggered right now, it'll revert back and block 100s of people out of the service with a 403. There is a story to play for this in the backlog to address the false positives

Reply:
Please check the comments from this ticket https://tools.hmcts.net/jira/browse/DIV-5332.
There were also some emails asking why do we have set the WAF on detection mode.

Reply:
Please see https://tools.hmcts.net/jira/browse/DIV-5724

Reply:
@qzhou-hmcts happy to go with this? I believe @magicobject is aware of https://tools.hmcts.net/jira/browse/DIV-5723 and we will prioritise it soon

Reply:
tbh, I don't know if there is a new plan to set the WAF in detection mode, last time I did the change many managers come asking the reasons for the change, maybe @magicobject or @yarpo  can clarify about this

Reply:
@qzhou-hmcts as you may be aware we're mandated to go back into prevention mode. See the #waf-detection-mode slack channel and DIV-5723